components/ipfs/ipfs_file_import_worker.cc

```diff
@@ -0,0 +1,98 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ipfs/ipfs_file_import_worker.h"
+
+#include <utility>
+
+#include "base/files/file_util.h"
+#include "base/guid.h"
+#include "base/task/post_task.h"
+#include "base/task_runner_util.h"
+#include "brave/components/ipfs/import_utils.h"
+#include "brave/components/ipfs/ipfs_constants.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/browser_thread.h"
+#include "net/base/mime_util.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "storage/browser/blob/blob_data_builder.h"
+#include "url/gurl.h"
+
+namespace {
+
+const char kFileValueName[] = "file";
```

Comment:
I believe this one is used in multiple places too, I meant in my previous review comment for all these to be moved and not just the first one only, so the comment applied to file and mime type strings too.